po/wesnoth/af.po

```diff
@@ -1751,11 +1751,11 @@ msgstr "Moet $unit.name die drietandvurk op tel?"
 
 #: data/core/macros/items.cfg:514
 msgid "storm trident^Take it"
-msgstr "storm drietandvurk^los dit"
+msgstr "los dit"
```

Comment:
Not an issue for this PR, but here "Take it" and "Leave it" are translated to the same phrase.

Reply:
Google says 'los dit' is 'leave it'. Clearing 'take it' as untranslated.